viewer/src/datamodels/pointcloud/PointCloudNode.ts

```diff
@@ -92,6 +92,28 @@ export class PointCloudNode extends THREE.Group {
     this._potreeNode.pointShape = value;
   }
 
+  /**
+   * Sets a visible filter on points of a given class.
+   * @param pointClass The point class type to which the filter should be applied. This can either
+   * be one of the well known classes from {@link WellKnownPointClassTypes} or a number for user defined
+   * classes.
+   * @param visible Boolean flag that determines if the point class type should be visible or not.
+   */
+  setClassVisible(pointClass: number | WellKnownPointClassTypes, visible: boolean): void {
```

Comment:
I suggest to name it like `setClassVisibility` because now it feels like a function that makes a certain class just visible, but it accepts visibility param. 

Or maybe it's just me 🤷‍♂️

Reply:
@maksnester : kind of agree, but what would you call `isClassVisible`?

Reply:
Also, maybe its just me from my days in .NET, but I feel Visibility should be an enum and not a bool. Leaning towards just keeping it, mainly because I think `isClassVisible` should match naming and that `isClassVisibility` doesn't make sense and that `getClassVisibility: boolean` seems a bit off to me.

Reply:
Agree that visibility doesn't sound well with a boolean param.